scripts/enemy.gd

```diff
@@ -50,15 +50,13 @@ func _process(delta):
 	var motion: Vector2
 	motion = move_and_slide(motion)
 
-	for i in range(get_slide_count()):
-		var collision = get_slide_collision(i)
-		var collider = collision.collider
-		if collider.is_in_group("Player"):  # Replace "AreaGroup" with your Area2D's group
-			# TODO:
-			# Handle collision with the Area2D
-			pass
-			# print("Colliding with Player")
-			# check if pogo is activated off of player node and don't do damage if it is
+	# for i in range(get_slide_count()):
```

Comment:
Maybe we get rid of this unless we think we'll need it later for smth

Reply:
yeah if we dont end up needing player collisions on the enemy kb i will remove